Formula/youtube-dl.rb

```diff
@@ -22,7 +14,7 @@ class YoutubeDl < Formula
   depends_on "rtmpdump" => :optional
```

Comment:
Can we remove `rtmpdump` here and treat it same as `ffmep` and `libav`, i.e. discuss it in `caveats`?